packages/slider/src/ux-slider.css

```diff
@@ -53,24 +49,24 @@ ux-slider>.slider-container>.slider-thumb-container>button {
   outline: 0;
 }
 
-ux-slider>.slider-container>.slider-thumb-container>button:hover, ux-slider>.slider-container>.slider-thumb-container>button:focus {
+.ux-slider__container--enabled .ux-slider__thumb-button:hover,
```

Comment:
This one is going to be nitpicky but its the last one I have. Move this rule down below `.ux-slider__container--disabled *`. That's the last change I have, everything else seems fine!

Reply:
I was not clear on the reason why, I apologize. It helps organize our CSS rules by CSS Specificity.

Reply:
Here is a handy tool to order CSS by specificity. 

https://jonassebastianohlsson.com/specificity-graph/

Reply:
ok should be done